oaknational/Oak-Web-Application
# feat: consolidate  capture event

## Description

- only one `$pageview` event
- changes `analyticsUseCase` and `pageName` properties to be single value (rather than list)
- see https://www.avo.app/schemas/5PhajbVijwhXVKIJtGMT/branches/a7OxEIjEp/diff
- make `viewType` page prop non optional

## Issue(s)

Fixes #1538 

## How to test

1. Go to {cloud link}
2. Click around
3. Check posthog live events, should see Pageview events coming through with custom properties: `linkUrl` `pageName` and `analyticsUseCase`
